src/Microsoft.ML.Data/Transforms/RowToRowTransformerBase.cs

```diff
@@ -23,7 +23,8 @@ protected RowToRowTransformerBase(IHost host)
             Host = host;
         }
 
-        public abstract void Save(ModelSaveContext ctx);
+        void ICanSaveModel.Save(ModelSaveContext ctx) => SaveModel(ctx);
+        private protected abstract void SaveModel(ModelSaveContext ctx);
```

Comment:
>private protected [](start = 8, length = 17)

I spent probably 10 mintues going through MSDN in attempt to understand how private protected which should be accessible only in this assembly works in OnnxTransform and other assemblies. And then I notice same question from Tom, but on top of file. Can you please stop force-pushing changes, it's screw ups codeflow in a bad way, and no way anyone will do codereview in web interface for 140 files.

Reply:
OK sounds good sorry about that!

---
In reply to: [255246374](https://github.com/dotnet/machinelearning/pull/2431#discussion_r255246374) [](ancestors = 255246374)